src/ripple/rpc/impl/ShardArchiveHandler.cpp

```diff
@@ -37,9 +37,9 @@ boost::filesystem::path
 ShardArchiveHandler::getDownloadDirectory(Config const& config)
 {
     return get(config.section(ConfigSection::shardDatabase()),
-               "download_path",
+               "partial_shard_download_path",
```

Comment:
Does this break existing config files?

Reply:
I guess that depends on what you mean by break. A server whose config file only provides the oudated property name (`download_path`) will use a default path as the `partial_download_path` which may or may not reflect whatever path was specified with the outdated name.

Reply:
Rolled back.